tests/acceptance/addon-smoke-test-slow.js

```diff
@@ -99,6 +100,68 @@ if (!experiments.MODULE_UNIFICATION) {
       expect(result.code).to.eql(0);
     }));
 
+    it('works when ember-cli-build is inside tests/dummy', co.wrap(function *() {
+      yield copyFixtureFiles('addon/ember-cli-build-at-tests');
+
+      let packageJsonPath = path.join(addonRoot, 'package.json');
+      let packageJson = fs.readJsonSync(packageJsonPath);
+
+      packageJson.dependencies = packageJson.dependencies || {};
+      // add HTMLBars for templates (generators do this automatically when components/templates are added)
+      packageJson.dependencies['ember-cli-htmlbars'] = 'latest';
+
+      fs.writeJsonSync(packageJsonPath, packageJson);
+
+      let result = yield runCommand('node_modules/ember-cli/bin/ember', 'build');
+
+      expect(result.code).to.eql(0);
+
+      let cssPath = path.join(addonRoot, 'addon', 'styles', 'app.css');
```

Comment:
Why check css for an ember-cli-build test?

Reply:
I wanted to ensure that the options of ember-cli-build are being respected(in this case, fingerprinting) when the file is placed at both locations. No specific reason as to why I chose app.css and not some other file.